src/scss/custom.scss

```
@@ -90,3 +90,8 @@ html {
 [data-theme="dark"] .docusaurus-highlight-code-line {
   background-color: rgb(0 0 0 / 30%);
 }
+
+html[data-theme="dark"] {
+  --search-local-highlight-color: #2d66c3;
```

Maybe we can keep on following OKP4 palette by using here main blue hexadecimal value:
```suggestion
  --search-local-highlight-color: #0f1e3d;
```